packages/application-components/src/components/internals/page-header-title.tsx

```diff
@@ -24,15 +23,10 @@ const SubtitleWrapper = styled.div`
 type TitleProps = Pick<Props, 'titleSize' | 'title' | 'truncate'>;
 
 const Title = (props: TitleProps) => {
-  const { theme } = useTheme();
```

Comment:
I actually changed some of these before running the codemod to use `themedValue` instead so it can modify them.